packages/cli/src/cmds/agentInstaller/javaBuildToolInstaller.ts

```diff
@@ -1,7 +1,30 @@
+import glob from 'glob';
+import os from 'os';
+import path from 'path';
 import chalk from 'chalk';
 import CommandStruct from './commandStruct';
 import { run } from './commandRunner';
 import { getOutput } from './commandUtil';
+import { findJetBrainsHome } from './jetBrainsSupport';
+
+export function addJetBrainsEnv() {
+  const jbHome = findJetBrainsHome();
+  if (!jbHome) {
+    return;
+  }
+
+  const javaHome = path.join(jbHome, 'jbr');
+  const gradleBin = path.join(jbHome, 'lib');
+  const mvnBin = path.join(jbHome, 'plugins/maven/maven3/bin');
+
+  // Make sure we don't override the user's settings: append to path, use the
+  // existing JAVA_HOME if it's set.
+  if (!process.env['JAVA_HOME']) {
+      process.env['JAVA_HOME'] = javaHome;
+  };
+  process.env['PATH'] = [process.env['PATH'], path.join(javaHome, 'bin'), gradleBin, mvnBin].join(path.delimiter);
```

Comment:
If an `.idea` directory is present, I think we'd want to reverse the order here and add the IntelliJ paths to the beginning of `PATH` so they're guaranteed to be picked up. Maybe we ask the user?

Reply:
This doesn't seem right to me. If the user has added a JDK to their `PATH`, they probably did it for a reason (e.g. they explicitly don't want to use the embedded JDK). 

What would be a case where we want to override what the user has done?